packages/elements/src/utils/fetcher/fetcher.ts

```diff
@@ -33,6 +34,10 @@ export const fetcher = async <T, B>({
   } as RequestInit)
 
   if (res.status < 400) {
+    if (returnRawResponse) {
```

Comment:
I saw your usage for this and I get it - so you can call res.blob() not res.json(). Would it make more sense to have a return type that defaults to json so the fetcher can return res.text, res.json, res.ok, res.blob depending on the param you pass in. You can then strongly type it using a generic type. Like I say, can default to JSON so it won't break anything. Think would be nice

Reply:
Why do you need to modify fetcher, I think if you have any other logic for this fetcher, pls write new one do not add a lot of logic to 1 function. Function only does 1 thing.